### DISCUSSION TITLE
Languagetools

### DISCUSSION THREAD

**Discussion by GregSchofield** [Zettlr/Zettlr#3378] [Ideas]
Noticed [_**LanguageTool Linter**_ for _Visual Studio Code_)](https://marketplace.visualstudio.com/items?itemName=davidlday.languagetool-linter) recognises markdown. This would be a great asset to Zettlr if it could be integrated. It runs on OSS (the Linux version of  _Visual Studio Code_), it is very good at picking up stupid mistakes which I am very prone to do.

**Comment by GregSchofield** [comment]
I will be testing https://github.com/aokellermann/markdown_lt, as a work around, perhps the author might be persuqded to better itergrate with Zettlr in the future asa plugin?

**Reply by GregSchofield** [reply]
Could not get the above to work, tried _Obsidian_ and was very pleased with the implementation there. **This should be added to Zettlr**.

**Comment by StefanBertels** [comment]
Maybe related (or subset): I would like to format code snippets within markdown code blocks -- at least if they have a valid language annotation. Could be just "right click on file node => run external command %1".

This could be very useful for JSON, SQL and many more.
